msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -5,62 +5,59 @@ description: Identify the meeting apps API references with examples and Code sam
-|**Meeting Details** | Enables you to get static meeting metadata. |**GET** _**/v1/meetings/{meetingId}**_| Bot SDK |
-|**shareAppContentToStage**| Enables you to share specific parts of the app to meeting stage from the app side panel in a meeting. |_**microsoftTeams.meeting.shareAppContentToStage((err, result) => {}, appContentUrl)**_|Microsoft Teams client SDK|
-|**getAppContentStageSharingState**| Enables you to fetch information about apps' sharing state on the meeting stage. |_**microsoftTeams.meeting.getAppContentStageSharingState((err, result)) => {}**_|Microsoft Teams client SDK|
-|**getAppContentStageSharingCapabilities**| Enables you to fetch the apps' capabilities for sharing to the meeting stage. |_**microsoftTeams.meeting.getAppContentStageSharingCapabilities((err, result)) => {}**_|Microsoft Teams client SDK|
-
-The following table provides the Bot Framework SDK methods for the APIs:
-
-|API|Bot Framework SDK method|
-|---|---|
-|**GetParticipant**| `GetMeetingParticipantAsync (Microsoft.Bot.Builder.ITurnContext turnContext, string meetingId = default, string participantId = default, string tenantId = default, System.Threading.CancellationToken cancellationToken = default);` |
-|**NotificationSignal** | `activity.TeamsNotifyUser(true, "https://teams.microsoft.com/l/bubble/APP_ID?url=&height=&width=&title=<title>&completionBotId=BOT_APP_ID");` |
-|**Meeting Details** | `TeamsMeetingInfo (string id = default);` |
+|Method| Description| Source|
+|---|---|----|
+|[**Get user context**](#getusercontext-api)| Get contextual information to display relevant content in a Teams tab.| Microsoft Teams client SDK|
+|[**Get participant**](#getparticipant-api)| Fetch participant information by meeting ID and participant ID. |Microsoft Bot Framework SDK|
+|[**Notification signal**](#notificationsignal-api)| Provide meeting signals using the existing conversation notification API for user-bot chat and allows to notify user action that shows an in-meeting dialog box. |Microsoft Bot Framework SDK|
+|[**Meeting details**](#meeting-details-api)| Get static meeting metadata. |Bot SDK |
+|[**Share app content to stage**](#shareappcontenttostage-api)| Share specific parts of the app to meeting stage from the app side panel in a meeting. |Microsoft Teams client SDK|
+|[**Get app content stage sharing state**](#getappcontentstagesharingstate-api)| Fetch information about apps' sharing state on the meeting stage. |Microsoft Teams client SDK|
+|[**Get app content stage sharing capabilities**](#getappcontentstagesharingcapabilities-api)| Fetch the apps' capabilities for sharing to the meeting stage. |Microsoft Teams client SDK|
+|[**Real-time Teams meeting events**](#real-time-teams-meeting-events)|Fetch real-time meeting events, such as actual start and end time.| Bot SDK|
 
 ## GetUserContext API
```

Comment:
All these headings should match the contents of the `Method` column letter for letter.

Reply:
Done